### PR TITLE
Chore: update variable names in lambda expressions

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Some small number of  braces/formatting are included.

### CODE REVIEW

File: src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java [hunk 1]
```diff
@@ -210,7 +210,9 @@ public void deleteAdmissionUpdated(AWTEvent e) {
 					//the patient is admitted
 					if (elemAdm.getId() == adm.getId())
 						//same admission --> delete
-						elem.setAdmission(null);	
+					{
```
Comment (mwithi) [inline src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java:213]: maybe bracket on same line of if?

**Review by mwithi — APPROVED** [review]
LGTM. Very good you differentiated the event types. Thanks a lot!

File: src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java]: I got carried away and fixed many more bracket/spacing issues.   Also found a few more names to change.

File: src/main/java/org/isf/utils/jobjects/JAgenda.java [hunk 2]
```diff
@@ -1274,7 +1296,9 @@ public void addElement(Object obj, int day) {
 		
 		tmpCalendar.set(Calendar.DAY_OF_MONTH, 1);
 		int firstDay = tmpCalendar.get(Calendar.DAY_OF_WEEK) - firstDayOfWeek;
-		if (firstDay < 0) firstDay+=7;
+		if (firstDay < 0) {
+			firstDay+=7;
```
Comment (mwithi) [inline src/main/java/org/isf/utils/jobjects/JAgenda.java:1300]: space between operand and operators

**Review by mwithi — APPROVED** [review]
LTGM. Thanks a lot!

File: src/main/java/org/isf/utils/jobjects/JAgenda.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/utils/jobjects/JAgenda.java]: fixed

**Pull request merged**